scrapers/RealityLovers.py

```diff
@@ -0,0 +1,195 @@
+    birthdate = time.strptime(
+        re.sub(r"(st|nd|rd|th)", r"", tree.xpath('//*[text()="Birthday:"]/../following-sibling::dd/text()')[0].strip()),
+        '%b %d %Y')
+    country = tree.xpath('//*[text()="Country:"]/../following-sibling::dd/text()')[0].strip()
+    measurements = tree.xpath('//*[text()="Cup:"]/../following-sibling::dd/text()')[0].strip()
+    height = re.sub(r".*\(([0-9]*) cm\)",
+                    r"\1",
+                    tree.xpath('//*[text()="Height:"]/../following-sibling::dd/text()')[0].strip())
+    weight = re.sub(r".*\(([0-9]*) kg\)",
+                    r"\1",
+                    tree.xpath('//*[text()="Weight:"]/../following-sibling::dd/text()')[0].strip())
+    socials = tree.xpath('//*/a[@class="girlDetails-socialIcon"]/@href')
+    tags = tree.xpath('//*/a[@itemprop="keyword"]/text()')
+    details = tree.xpath('//*/p[@class="girlDetails-bioText"]/text()')[0].strip()
+
+    return {
+        "Name": name,
+        # making some assumptions here
+        "Gender": "transgender_female" if re.match(r'.*tsvirtuallovers.*', inp['url'], re.IGNORECASE) else "female",
+        "URL": inp['url'],
+        "Twitter": next(social for social in socials if re.match(r'.*twitter.*', social, re.IGNORECASE)),
+        "Instagram": next(social for social in socials if re.match(r'.*instagram.*', social, re.IGNORECASE)),
+        "Birthdate": time.strftime("%Y-%m-%d", birthdate),
+        "Country": country,
+        "Height": height,
+        "Weight": weight,
+        "Measurements": measurements,
+        "Tags": [{
+            'name': x
+        } for x in tags],
+        "Image": main_image_src,
+        "Details": details,
+    }
+
+
+def sceneByURL():
+    # read the input.  A URL must be passed in for the sceneByURL call
+    inp = json.loads(sys.stdin.read())
+    if not inp['url']:
+        log.error('No URL Entered')
+
+    # get the url specified in the input and validate the response code
+    scraped = session.get(inp['url'])
+    if scraped.status_code >= 400:
+        log.error('HTTP Error: %s' % scraped.status_code)
+    log.trace('Scraped the url: ' + inp["url"])
+
+    # get the data we can scrape directly from the page
+    tree = html.fromstring(scraped.content)
+    title = "".join(tree.xpath('//*[@class="video-detail-name"]/text()')).strip()
+    details = re.sub(r'\n\s*\n', '\n\n', "\n\n".join(tree.xpath('//*[@itemprop="description"]//text()')).strip())
+    images = tree.xpath('//*[@itemprop="thumbnail"]/@data-big')
+    image_url = ""
+    if len(images) > 0:
+        image_url = re.sub(r' .*', r'', images[0])
+    tags = tree.xpath('//a[@itemprop="keyword"]/text()')
+    actors = tree.xpath('//a[@itemprop="actor"]/text()')
+
+    # create our output
+    return {
+        'title': title,
+        'tags': [{
+            'name': x
+        } for x in tags],
+        'details': details,
+        'image': image_url,
+        'studio': {
+            'name': "Reality Lovers"
```

Comment:
you are using the scraper for tsvirtuallovers as well

getting the studio name as
```python
    studio = tree.xpath('//span[@class="icon header-logo"]/text()')[0]
```
and using it here works for both sites